sister/download.py

```diff
@@ -0,0 +1,77 @@
+import tempfile
+import contextlib
+import hashlib
+import os
+import shutil
+import sys
+import zipfile
+from urllib import request
+from pathlib import Path
+
+
+_cache_root = os.environ.get(
+    'LINEFLOW_CACHE_ROOT',
```

Comment:
I think this line should be `SISTER_CACHE_ROOT`.

Reply:
Hehe, That's very true.
Thank you!